OpenRA.Mods.Common/Traits/Cloak.cs

```diff
@@ -161,7 +161,8 @@ IEnumerable<IRenderable> IRenderModifier.ModifyRender(Actor self, WorldRenderer
 
 			if (Cloaked && IsVisible(self, self.World.RenderPlayer))
 			{
-				var palette = string.IsNullOrEmpty(Info.Palette) ? null : Info.IsPlayerPalette ? wr.Palette(Info.Palette + self.Owner.InternalName) : wr.Palette(Info.Palette);
+				var palette = wr.Palette(Info.IsPlayerPalette ? Info.Palette + self.Owner.InternalName : Info.Palette);
```

Comment:
If `Info.Palette` is `null` than this can crash now.

Reply:
You mean `Info.Palette + self.Owner.InternalName` ? No, it can't, cause `null + string` is `string`. Weird, I know.
![csharp](https://github.com/OpenRA/OpenRA/assets/119738087/b5b21101-7ee7-46e3-a9c3-ba990005df37)

Reply:
Huh ok, I thought that would crash. Good to know. ^^